packages/surveys/src/components/wrappers/auto-close-wrapper.tsx

```
@@ -62,13 +62,23 @@ export function AutoCloseWrapper({
-      ) : null}
-      <div onClick={stopCountdown} onMouseOver={stopCountdown} className="fb-h-full fb-w-full">
+    <div className="fb-h-full fb-w-full fb-flex fb-flex-col">
+      <div
+        onClick={stopCountdown}
+        onMouseOver={stopCountdown} // NOSONAR // We can't check for onFocus because the survey is auto focused after the first question and we don't want to stop the countdown
+        role="button"
+        className="fb-h-full fb-w-full"
+        data-testid="fb__surveys__auto-close-wrapper-test"
+        onKeyDown={stopCountdown}
+        aria-label="Auto close wrapper"
+        onTouchStart={stopCountdown}>
```

Biome (2.1.2)

[error] The HTML element with the interactive role "button" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "button".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)